src/sagemaker/workflow/steps.py

```diff
@@ -93,6 +93,26 @@ def ref(self) -> Dict[str, str]:
         return {"Name": self.name}
 
 
+@attr.s
+class CacheConfig:
+    """Step to cache pipeline workflow.
```

Comment:
This is not so much a "Step" but a configuration. Can you update the description?

Reply:
sure.